contracts/src/TeleporterTokenSource.sol

```diff
@@ -277,12 +291,14 @@ abstract contract TeleporterTokenSource is
                 return;
             }
         } else {
-            adjustedAmount = _prepareSend(
-                input.destinationBlockchainID,
-                input.destinationBridgeAddress,
-                amount,
-                input.primaryFee
-            );
+            (adjustedAmount, feeAmount) = _prepareSend({
```

Comment:
We don't actually use the returned `feeAmount` in this function. Should we be initializing `feeAmount = input.primaryFee` and using `feeAmount` later in the call to `_sendTeleporterMessage`?

Reply:
Yeah, you're right. Good catch :)